<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SocketClient</class>
 <widget class="QWidget" name="SocketClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SocketClient</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>服务端IP：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>70</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>服务端端口号：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>发送内容：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>接收数据：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="IPlineEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="PortlineEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="sendtextEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>120</y>
     <width>141</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="recvtextEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>210</y>
     <width>141</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="connectBtn">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>30</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>连接服务端</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sendBtn">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>90</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
